xAPI-Data.md

```diff
@@ -697,14 +703,14 @@ Some examples:
 
 * <a name="2.4.4.s1.b2"></a>The Object is an Agent: "Nellie interviewed Jeff."
 
-* <a name="2.4.4.s1.b3"></a>The Object is a SubStatement or Statement Reference (different implementations, but similar when human-read): 
-"Nellie commented on 'Jeff wrote an essay about hiking.'"
+* <a name="2.4.4.s1.b3"></a>The Object is a SubStatement or Statement Reference (different implementations, but similar 
+when human-read): "Nellie commented on 'Jeff wrote an essay about hiking.'"
 
 ###### <a name="2.4.4.s2"></a>Details
 
 Objects which are provided as a value for this property SHOULD include an "objectType" 
-property. If not specified, the objectType is assumed to be "Activity". Other valid values 
-are: [Agent](#agentasobj), [Group](#agentasobj), [SubStatement](#substmt) or [StatementRef](#stmtref).
+property. If not specified, the objectType is assumed to be `Activity`. Other valid values 
+are: `[Agent](#agentasobj)`, `[Group](#agentasobj)`, `[SubStatement](#substmt)` or `[StatementRef](#stmtref)`.
```

Comment:
I did find the right way to make this work, but it looks bad too.  Taking out the links as 1) Activity already doesn't have one and 2) This doesn't seem like the place where people using the spec should use as a jumping spot to learn about all of the different objectTypes.  They can use the TOC or simply keep reading.